### PR TITLE
Exposing generated multitest_utils via "mt" feature

### PR DESCRIPTION
**Pull request by jawoznia**

### CODE REVIEW

File: sylvia-derive/src/multitest.rs [hunk 1]
```diff
@@ -268,18 +268,18 @@ impl<'a> MultitestHelpers<'a> {
             .collect();
 
         quote! {
-            #[cfg(test)]
+            #[cfg(any(test, feature = "mt"))]
```
Comment (hashedone) [inline sylvia-derive/src/multitest.rs:271]: There should be no cfg attribute generated at all. Generation of the whole chunk should be enabled/disabled based on the state of sylvia feature flag. Here you assume you have any control about feature of crate using your toolbox which is a mistake. Just remove the whole `cfg` and make sure it is not generated when `mt` feature is not enabled on `syvia-derive`.

**Review by hashedone — COMMENTED** [review]
One little problem commented, besides that it is good.

**Review by hashedone — APPROVED** [review]
LGTM

**Comment by jawoznia** [comment]
Closing due to #88 being delivered

**Pull request closed without merging**